src/main/java/frc/robot/commands/sequences/BlueBalanceAuton.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.commands.sequences;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.util.Units;
+
+import frc.robot.subsystems.RollerSubsystem;
+import frc.robot.subsystems.TiltedElevatorSubsystem;
+import frc.robot.subsystems.drivetrain.BaseSwerveSubsystem;
+
+/**
+ * The middle auton sequence, where the robot deposits its preloaded game piece and balances
+ * on the charging station.
+ */
+public class BlueBalanceAuton extends BaseAutonSequence {
+    private static final Pose2d INITIAL_POSE = new Pose2d(
+        Units.inchesToMeters(PlacePosition.BLUE_INIT_X_IN),
+        Units.inchesToMeters(107.638),
+        Rotation2d.fromDegrees(180)
+    );
+    private static final Pose2d OUTSIDE_POSE = new Pose2d(
```

Comment:
is it possible for one of these (e.g. red/blue) be in a mirrored relationship with the other? this'll make it easy to tweak both during competitoin